htdocs/user/card.php

```diff
@@ -548,7 +548,7 @@
 					$ret = $object->update($user);		// This may include call to setPassword if password has changed
 					if ($ret < 0) {
 						$error++;
-						if ($db->errno() == 'DB_ERROR_RECORD_ALREADY_EXISTS') {
+						if ($db->errno() == 'ErrorUpdateCanceledDueToDuplicatedUniqueValue') {
```

Comment:
This is surely not the error code. Should be the error message...

Reply:
sorry, too fast on the correction :)